### PR TITLE
[FEAT]Lesson-10-task-3

### PR DESCRIPTION
**Pull request by heilrake**
image

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import './App.css';
 
-import CurrentTask from './lesson/Lesson10/Task1/Index';
+import CurrentTask from './lesson/Lesson10/Task3/Index';
```
Comment (daviddubliakov) [inline src/App.jsx]: don't use Index

File: src/lesson/Lesson10/Task3/component/TableOfUsers.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+          <TableCell colSpan='2'>Age</TableCell>
+        </TableRow>
+      </TableHead>
+      <TableBody>
+        {users.map((item, index) => (
+          <TableRow key={index}>
+            <TableCell>{item.firstName}</TableCell>
+            <TableCell>{item.lastName}</TableCell>
+            <TableCell>{item.age}</TableCell>
+            <TableCell><Link href='#' onClick={() => showMessage(item.firstName)}>Click me!</Link ></TableCell>
```
Comment (daviddubliakov) [inline src/lesson/Lesson10/Task3/component/TableOfUsers.jsx]: format this line

**Review by daviddubliakov — CHANGES_REQUESTED** [review]

File: src/lesson/Lesson10/Task3/component/TableOfUsers.jsx (hunk 2 above)
Comment (AI-Avanger) [reply src/lesson/Lesson10/Task3/component/TableOfUsers.jsx]: fixed

File: src/App.jsx (hunk 1 above)
Comment (AI-Avanger) [reply src/App.jsx]: fixed

**Pull request merged**